google/cloud/spanner/row.h

```diff
@@ -272,9 +272,16 @@ class Row {
   std::tuple<Types...> values_;
 };
 
-// Returns the row-element at column `I`. This function will be found via ADL
-// to make `Row<Ts...>` work with `internal::ForEach`. Users should not call
-// this function directly; instead, simply call `row.get<I>()`.
+/**
+ * Returns the row-element at column `I`.
+ *
+ * @note Library users should not need to call this function directly; instead,
```

Comment:
This is really not intended to be part of the public API. It exists in this namespace because ADL requires that. I'm concerned that documenting like this makes it look like a public part, but it's not.

Does doxygen give us any way to *remove* a function from the generated docs?

Reply:
> 
> Does doxygen give us any way to _remove_ a function from the generated docs?

Yes. I added `GetElement` to the excluded symbols.  I kept the doxygen comments because I want (someday) to generate a "developer's version" of the Doxygen docs.

> This is really not intended to be part of the public API.

We may not want it to be, but it *is* part of the public API.  It is in the same namespace we put all the public functions.